BrowserKit/Tests/TabDataStoreTests/TabDataStoreTests.swift

```diff
@@ -2,138 +2,138 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
-import XCTest
-@testable import TabDataStore
-import Common
-
-final class TabDataStoreTests: XCTestCase {
```

Comment:
I've disabled the tests in this file to unblock this PR as they are causing problems and will be addressed in this PR https://github.com/mozilla-mobile/firefox-ios/pull/14057
cc @petruSt24

Reply:
@OrlaM is this not part of a suite that you have to comment out the whole file instead of skipping it? 🤔 

<img width="507" alt="image" src="https://user-images.githubusercontent.com/8919439/233720244-810078be-80a3-482c-87b7-8cc69f8cb170.png">

Fine with either approach but curious